### ISSUE TITLE
Support for Xcode 15 accessibility audits

### ISSUE THREAD

**Issue by erichoracek** [EmergeTools/SnapshotPreviews-iOS#29]
Looks like Xcode 15 supports triggering [accessibility audits](https://www.polpiella.dev/xcode-15-automated-accessibility-audits) from XCUI tests, it would be cool to support running an accessibility audit for each of your snapshots.

**Comment by noahsmartin** [comment]
Good idea! I looked into this a bit and `performAccessibilityAudit` can be passed a block to handle the resulting issue, however the issue is a `XCUIAccessibilityAuditIssue` and not a `XCTIssue`. This makes it tricky to record the issue outside of the test case running when we call `performAccessibilityAudit`. I think this will require a bit of a refactor to get the app put into a testable state within each test function, rather than the current structure of generating all the images before we create the test functions

**Comment by noahsmartin** [comment]
This is now implemented in https://github.com/EmergeTools/SnapshotPreviews-iOS/pull/42 thanks for suggesting!
